types/mds.d.ts

```diff
@@ -36,6 +36,8 @@ export interface Device {
 
 export type DeviceID = Pick<Device, 'provider_id' | 'device_id'>
 
+export type VehicleEventPrimaryKey = { timestamp: Timestamp; device_id: UUID } | null
```

Comment:
I think this should be in types for `mds-db`

Reply:
I could move it, but it made sense to define it beside where `VehicleEvent` is declared.  Much of the stuff in `mds.d.ts` should move to `mds-db`, but @janedotx says there's a circular dependency between `mds-utils` and `mds-db` that needs to be resolved for that to happen. That should be a separate refactor to assess what's going on there.